packages/react-admin-core/src/table/LocalChangesToolbar.tsx

```diff
@@ -19,9 +20,18 @@ export class TableLocalChangesToolbar extends React.Component<IProps> {
                 {!this.props.loading && (
                     <Toolbar>
                         <Button color="default" onClick={this.handleSaveClick} startIcon={<SaveIcon />}>
-                            Speichern
+                            <FormattedMessage
+                                id="reactAdmin.core.table.localChangesToolbar.save"
+                                defaultMessage="Speichern"
+                                description="Speichern Button"
+                            />
                         </Button>
-                        {this.props.localChangesCount} unsaved change(s)
+                        <FormattedMessage
+                            values={{ count: this.props.localChangesCount }}
+                            id="reactAdmin.core.table.localChangesToolbar.unsavedItems"
+                            defaultMessage="{count, plural, =0 {no unsaved changes} one {# unsaved change} other {# unsaved changes}}"
```

Comment:
The defaultMessage should be German.

Reply:
no, english please. The only reason why it is currently german is, because we didn't have translations.

And please don't use this description, except it is really needed.

Reply:
It's probably better to keep it german. This PR is meant to localize the core only. This PR should not be dependent on the actual translations. 
The default language should be changed in the PR where the actual translations are available.

Reply:
ok...